src/sonic-db-cli/Makefile.am

```diff
@@ -0,0 +1,24 @@
+###########################################################################
+##
+## File:        ./Makefile.am
+## Versions:    $Id: Makefile.am,v 1.0 2022/04/02 12:04:29 liuh@microsoft.com Exp $
+## Created:     2022/04/02
+##
+###########################################################################
+ACLOCAL_AMFLAGS = -I m4 --install
+ACLOCAL_AMFLAGS = -I config
+AUTOMAKE_OPTIONS = subdir-objects
+SUBDIRS = tests
+
+bin_PROGRAMS = sonic-db-cli
```

Comment:
>sonic-db-cli [](http://example.com/codeflow?start=15&length=12)

Offline discussed. Let's move this application to sonic-swss-common repo. Following original practice that in sonic-py-swsssdk repo.

Reply:
I will close this PR and send another PR for this.